dispute_gateway.go

```diff
@@ -2,12 +2,69 @@ package braintree
 
 import (
 	"context"
+	"encoding/xml"
+	"fmt"
+	"math"
 )
 
 type DisputeGateway struct {
 	*Braintree
 }
 
+func (g *DisputeGateway) fetchDisputes(ctx context.Context, query *SearchQuery, page int) (*DisputeSearchResult, error) {
+	resp, err := g.executeVersion(ctx, "POST", fmt.Sprintf("disputes/advanced_search?page=%d", page), query, apiVersion4)
+	if err != nil {
+		return nil, err
+	}
+	var v struct {
+		CurrentPageNumber int        `xml:"current-page-number"`
+		PageSize          int        `xml:"page-size"`
+		TotalItems        int        `xml:"total-items"`
+		XMLName           string     `xml:"disputes"`
+		Disputes          []*Dispute `xml:"dispute"`
+	}
+	err = xml.Unmarshal(resp.Body, &v)
+	if err != nil {
+		return nil, err
+	}
+	pageCount := float64(v.TotalItems) / float64(v.PageSize)
+	if math.Mod(pageCount, 1) != 0 {
+		pageCount++
+	}
```

Comment:
This seems to be aiming to implement `math.Ceil`, doesn't it?

```suggestion
	pageCount := math.Ceil(float64(v.TotalItems) / float64(v.PageSize))
```